### ISSUE TITLE
Instagram: Link tagged people properly when they're not mentioned in the caption.

### ISSUE THREAD

**Issue by beaulebens** [beaulebens/keyring-social-importers#29]
If you're using the Instagram importer and the People & Places plugin, then you'd expect that if you tagged a person in a photo, they'd be tagged as a "Person" on the imported post. Instead, this will only work if you mention them via username in the caption. It might also work if they already exist, but new People are not created properly.

In this photo; https://www.instagram.com/p/BsFVOCmgBbk/ there are multiple people mentioned, and they show up in the API response as follows:

```
[users_in_photo] => Array
        (
            [0] => stdClass Object
                (
                    [user] => stdClass Object
                        (
                            [username] => akires
                        )

                    [position] => stdClass Object
                        (
                            [x] => 0.5342222222
                            [y] => 0.3244444444
                        )

                )

            [1] => stdClass Object
                (
                    [user] => stdClass Object
                        (
                            [username] => dewizzzle
                        )

                    [position] => stdClass Object
                        (
                            [x] => 0.4542222222
                            [y] => 0.4684444444
                        )

                )

            [2] => stdClass Object
                (
                    [user] => stdClass Object
                        (
                            [username] => makethingsdaily
                        )

                    [position] => stdClass Object
                        (
                            [x] => 0.8045542895
                            [y] => 0.3795555556
                        )

                )

            [3] => stdClass Object
                (
                    [user] => stdClass Object
                        (
                            [username] => taurean_chic
                        )

                    [position] => stdClass Object
                        (
                            [x] => 0.6902057778
                            [y] => 0.3804444444
                        )

                )

        )
```

but on the imported post, only `akires` registers (previously created).

[This block of code](https://github.com/beaulebens/keyring-social-importers/blob/master/importers/keyring-importer-instagram.php#L196:L204) might not be catching them properly, or [this block might not be creating new entries properly](https://github.com/beaulebens/keyring-social-importers/blob/master/importers/keyring-importer-instagram.php#L330:L337) (perhaps because we only get a username for them?).

**Comment by beaulebens** [comment]
See also; [where the People entries are created](https://github.com/beaulebens/people-places/blob/master/people-places.php#L477:L520) in the People & Places plugin.

**Comment by beaulebens** [comment]
Looks like it is probably because [I'm looking for a `full_name` property](https://github.com/beaulebens/keyring-social-importers/blob/master/importers/keyring-importer-instagram.php#L199) to pass as `name`, but in this data we only have a `username`.

**Comment by beaulebens** [comment]
Fixed in a083e3207e95b93aeb47e928852f2a1b13ec2b43 (in the main importer, and in the existing reprocessor, so you can just run the "People tagged or mentioned on Instagram" reprocessor to clean up existing posts).
